examples/index.json

```diff
@@ -1539,6 +1539,73 @@
             }
         ]
     },
+    {
+        "title": "Testing",
+        "column": 3,
+        "category": "Language concepts",
```

Comment:
Does Testing fall under the `Language Concepts` category still or would it change with the new structure? 

@praneesha

Reply:
Im just checking since there is a tilte called `Deployment` that has the category "Language Concepts" but its falling under "Common Libraries" in the website. 

I think its because its positioned towards the end of the index.json file

Reply:
Moved it to the correct spot along with Deployment